_std/defines/item.dm

```diff
@@ -194,6 +194,7 @@
 #define W_CLASS_BULKY 4
 #define W_CLASS_HUGE 5
 #define W_CLASS_GIGANTIC 6
+#define W_CLASS_UNSTORABLE 7
```

Comment:
this will have odd side-effects, such as monkeys not picking up anything of this class, and TK pull not working on them either. I don't think that this is a good approach - consider one of the flags vars instead?